syncano/models/fields.py

```diff
@@ -687,10 +686,33 @@ def to_native(self, value):
         return value
 
 
-GeoPointStruct = namedtuple('GeoPointHelper', ['latitude', 'longitude'])
+class GeoPoint(Field):
 
+    KILOMETERS = '_in_kilometers'
+    MILES = '_in_miles'
 
-class GeoPoint(Field):
+    class GeoPointStruct(object):
+
+        def __init__(self, latitude=None, longitude=None, distance=None, unit=None):
```

Comment:
Also if latitude and longitude are required they should not have default values.